app/helpers/waste_exemptions_engine/edit_helper.rb

```diff
@@ -2,6 +2,8 @@
 
 module WasteExemptionsEngine
   module EditHelper
+    LOCALE = "waste_exemptions_engine.edit_forms"
```

Comment:
Why does the i18n scope need to be added explicitly?

Reply:
It's referenced a couple times in later on so just defining it here. What would you suggest instead?

Reply:
We don't need to state the scope explicitly elsewhere, wondering why that's needed here. Is it because it's a helper?

Reply:
if you look further down I make use of the LOCALE in the call to i18n - given the scope of the helper is only for edit forms I've defined the scope at the top so it can be used wherever the i18n is called. If it looks odd I can just repeatedly write `waste_exemptions_engine.edit_forms` in the `t()` calls instead?

Reply:
@PaulDoyle-DEFRA this is resolved now - I can leave off the prefix but need to add it in the tests like so: https://github.com/DEFRA/waste-exemptions-engine/pull/671/files#diff-b8714c1d7299d1d3d6b05c2bda31f96e36cc9b25e996aadbbbc1a978fae7a34aR7